package.json

```diff
@@ -687,7 +687,7 @@
     "copy-webpack-plugin": "^6.0.2",
     "cpy": "^8.1.1",
     "css-loader": "^3.4.2",
-    "cssnano": "^4.1.11",
+    "cssnano": "5.0.0",
```

Comment:
Since it was a major version bump, I was concerned with pushing it too far forward; from 4.1.11 to 5.0.0, the [only breaking changes](https://cssnano.co/docs/changelog/) were:
- minimum supported postcss version is 8.2.1
- minimum require version of node is 10.13

The changes would not affect KB since we are well past Node 10 and we currently use `postcss` `^8.4.14`

Reply:
Yeah, my suggestion was just `"5.0.0"` ---> `"^5.0.0"` to be consistent with other dependency declarations we have in `package.json`.